core-blocks/image/index.js

```diff
@@ -101,10 +101,13 @@ export const settings = {
 				isMatch: ( node ) => node.nodeName === 'FIGURE' && !! node.querySelector( 'img' ),
 				schema,
 				transform: ( node ) => {
-					const matches = /align(left|center|right)/.exec( node.className );
-					const align = matches ? matches[ 1 ] : undefined;
+					const className = node.className + ' ' + node.querySelector( 'img' ).className;
```

Comment:
What is being done here with the concatenation?

Can you add an inline comment?

Reply:
Yes, I'll add a comment. The class we need here is on the image, and for alignment it would be good to look at both the image and figure. Doesn't hurt.